persistence/src/main/java/org/mskcc/smile/persistence/neo4j/TempoRepository.java

```diff
@@ -82,4 +83,42 @@ Tempo mergeMafCompleteEventBySamplePrimaryId(@Param("primaryId") String primaryI
             + "->(s:Sample)-[:HAS_METADATA]->(sm: SampleMetadata {primaryId: $primaryId}) "
             + "RETURN cc.date ORDER BY cc.date ASC LIMIT 1")
     String findInitialPipelineRunDateBySamplePrimaryId(@Param("primaryId") String primaryId);
+
+    @Query("MATCH (t:Tempo {accessLevel: 'MSK Embargo'}) "
+            + "WHERE date(t.embargoDate) < date() "
+            + "RETURN t.smileTempoId")
+    List<UUID> findTempoIdsNoLongerEmbargoed();
```

Comment:
this was only ever used to find tempo ids no longer under embargo so I updated this to hard code that query instead of having to pass it along as a param